### PR TITLE
Add notes on kubeadm clusters version

### PR DESCRIPTION
**Pull request by pegasas**
~resolves~ helps with https://github.com/kubernetes/website/issues/44674

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ please refer to following pages instead:
 - [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -4 >}} to {{< skew currentVersionAddMinor -3 >}}](https://v{{< skew currentVersionAddMinor -3 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -5 >}} to {{< skew currentVersionAddMinor -4 >}}](https://v{{< skew currentVersionAddMinor -4 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 
+{{< note >}}
+it's recommended to match your kubelet and kubeadm versions, but you can also use an older kubelet version.
+For more details, please visit [kubeadm's skew against the kubelet](docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubelet)
```
Comment (dipesh-rawat) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:26]: Few corrections might be needed.
```suggestion
It's recommended to match your kubelet and kubeadm versions, but you can also use an older kubelet version.
For more details, please visit [kubeadm's skew against the kubelet](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubelet).
```

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ please refer to following pages instead:
 - [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -4 >}} to {{< skew currentVersionAddMinor -3 >}}](https://v{{< skew currentVersionAddMinor -3 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading a kubeadm cluster from {{< skew currentVersionAddMinor -5 >}} to {{< skew currentVersionAddMinor -4 >}}](https://v{{< skew currentVersionAddMinor -4 "-" >}}.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 
+{{< note >}}
+It's recommended to match your kubelet and kubeadm versions, but you can also use an older kubelet version.
+For more details, please visit [kubeadm's skew against the kubelet](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubelet).
+{{< /note >}}
+
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:28]: actually, i know i said to add this as a note, but can you please move it under Additional information.
i forgot this page has such a section. just drop the `note` tags and place it as a simple bullet point, second in position after this one  `- The instructions below outline when to ...`

**Review by neolit123 — COMMENTED** [review]
thanks for the PR!

added comment about a better location for the note.
also please change the OP to say `fixes <issue-number>`

**Review by neolit123 — COMMENTED** [review]
please squash the commits to 1
then LGTM

**Comment by pegasas** [comment]
> please squash the commits to 1 then i can LGTM

Thank you @neolit123 .
I've squash the commits to 1

**Comment by pegasas** [comment]
> please squash the commits to 1 then i can LGTM

Thank you @neolit123 .
I've squash the commits to 1

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ The upgrade workflow at high level is the following:
   first drain the node (or nodes) that you are upgrading. In the case of control plane nodes,
   they could be running CoreDNS Pods or other critical workloads. For more information see
   [Draining nodes](/docs/tasks/administer-cluster/safely-drain-node/).
+- It's recommended to match your kubelet and kubeadm versions, but you can also use an older kubelet version.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:45]:
```suggestion
- The Kubernetes project recommends that you match your kubelet and kubeadm versions.
  You can instead use an a version of kubelet that is older than kubeadm, provided it is within the
  range of supported versions.
```

**Comment by kfox1111** [comment]
Is the recommendation to use the same version of kubeadm for your kubelet as the kubelet your upgrading to, or the kubeadm the control plane is running?

Was seeking clarity in the docs about node versions vs control plane versions. with the new skew policy.

**Comment by sftim** [comment]
The control plane doesn't “run” kubeadm in the sense of a continous process so the question in https://github.com/kubernetes/website/pull/44683#issuecomment-1883205579 is hard to answer, but the observation makes sense overall:
could someone misinterpret this to infer that they can use a kubeadm to set up a kubelet that is not within supported version skew?

If they could, let's mitigate that.

I don't think this PR is making things worse than they are in the published website, so:
/lgtm

**Comment by sftim** [comment]
I edited the [description](https://github.com/kubernetes/website/pull/44683#issue-2072152928).

**Comment by neolit123** [comment]
> Is the recommendation to use the same version of kubeadm for your kubelet as the kubelet your upgrading to, or the kubeadm the control plane is running?

the process is - upgrade kubeadm first and then kubelet.
but kubeadm manages the kubelet settings, thus the kubelet version should ideally match the kubeadm version, that was just used to upgrade.

same for CP and worker nodes.

**Review by Princesso — COMMENTED** [review]
/lgtm

**Pull request merged**